### ISSUE TITLE
New danger rules to add

### ISSUE THREAD

**Issue by jessesquires** [jessesquires/JSQDataSourcesKit#122]
After #107 is complete. 

```ruby
# Milestones are required to track what's included in each release
if has_source_changes
  has_milestone = !github.pr_json['milestone'].nil?
  warn('All pull requests should have a milestone attached.', sticky: false) unless has_milestone
end

# Docs are regenerated when releasing
has_doc_changes = !git.modified_files.grep(/docs\//).empty?
has_doc_gen_title = github.pr_title.include? "#docgen"
if has_doc_changes && !has_doc_gen_title
  fail("Docs are regenerated when creating new releases.")
  message("Docs are generated by using [Jazzy](https://github.com/realm/jazzy). If you want to contribute, please update the markdown files or doc comments directly.")
end

```

**Comment by kevnm67** [comment]
@jessesquires Do you want to include this in #120? I have these changes stashed ;)

**Comment by jessesquires** [comment]
@kevnm67 sure! i'm ok with that

**Comment by jessesquires** [comment]
@kevnm67 actually, let's wait and do this in a follow-up PR after we merge. See my comments on #120
